### PR TITLE
Add HelpCenter documentation

### PR DESCRIPTION
**Pull request by gtema**
- rework docsportal docs and introduce dedicated HelpCenter doc sharing
  some commong parts
- add diagrams

### CODE REVIEW

File: doc/source/docsportal_sec.rst.inc [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+Interface Description
+---------------------
+
+The only public facing interface is the regular Web using HTTPS (automatic
+forwarding from HTTP).
+
+Tenant Security
+---------------
+
+Documentation Portal is not providing any tenant support. All documents that are hosted
```
Comment (vladimirhasko) [inline doc/source/docsportal_sec.rst.inc:88]: What you mean by Documentation portal is not providing any tenant support?
Documentation portal is basically support for customers just by its definition.
Maybe only some rephrasing is needed or better explanation

File: doc/source/docsportal_sec.rst.inc [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+Patch Management
+----------------
+
+The service consists of OpenSource elements only. Whenever new release of any
+software element (haproxy, nginx, zuul) is identified a Pull Request to this
+repository need to be created to update the software. Pathing of the
+underlaying VM (haproxy) is executed as a regular job applying all the existing
+OS updates.
+
+Hardening
```
Comment (vladimirhasko) [inline doc/source/docsportal_sec.rst.inc:122]: Maybe some reference to Deutsche Telekom Hardening policies?

File: doc/source/docsportal_sec.rst.inc [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+---------
+
+All configuration files for the hosts, Cloud Load Balancer configuration and K8
+configuration is part of this repository. Every VM is managed by the System
+Config project applying the same hardening rules to evenry host according to
+the configuration.
+
+Web server is deployed as a Kubernetes pod in a dedicated namespace and receives an Ingress.
```
Comment (vladimirhasko) [inline doc/source/docsportal_sec.rst.inc:130]: not sure if this belongs to hardening

File: doc/source/docsportal_sec.rst.inc [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+
+Certificate Handling
+--------------------
+
+SSL Certificates are obtained using Let's Encrypt Certificate authority.
+Following is important:
+
+* Certificate for the K8 deployment can managed by the
```
Comment (vladimirhasko) [inline doc/source/docsportal_sec.rst.inc:138]: can be managed by

File: doc/source/docsportal_sec.rst.inc [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+  `CertManager <https://cert-manager.io/>`_ deployed in
+  the Kubernetes cluster. This is achieved by placing
+  Kubernetes annotation on the deployment.
+* Alternatively SSL Certificate for the K8 installation may be generated on the
+  deployment server and provided into the K8 as secrets.
+* Certificates for the other involved components (Zuul,
+  Swift) are managed by the corresponding components
+  themselves.
+
+Backup and Restore
```
Comment (vladimirhasko) [inline doc/source/docsportal_sec.rst.inc:148]: basically you can mention that all data are stored in their source form in github with all the history modifications as well.
So it's simple to restore documentation to specific time in past if needed by rebuilding whole documentation to that point.

File: doc/source/docsportal_sec.rst.inc (hunk 2 above)
Comment (gtema) [reply doc/source/docsportal_sec.rst.inc:123]: Added

File: doc/source/docsportal_sec.rst.inc (hunk 5 above)
Comment (gtema) [reply doc/source/docsportal_sec.rst.inc:148]: done

**Review by vladimirhasko — APPROVED** [review]

**Pull request merged**